feat(ButtonGroup): add `orientation` prop (#603)
nuxtlabs/ui: docs/content/2.elements/5.button.md

To stack buttons as a group, use the `ButtonGroup` component.

- To size all the buttons equally, pass the `size` prop

> - To change the orientation all the buttons equally, pass the `orientation` prop to `vertical`

Reviewer: ```suggestion
- To change the orientation of the buttons, set the `orientation` prop to `vertical`
```
